teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -145,55 +285,79 @@ interface Branch {
     val isDefault: Boolean
 }
 
+interface BuildCanceledInfo {
+    val user: User?
+    val cancelDate: Date
+}
+
 interface Build {
     val id: BuildId
-    val buildTypeId: BuildConfigurationId
-    val buildNumber: String
-    val status: BuildStatus
+    val buildConfigurationId: BuildConfigurationId
+    val buildNumber: String?
+    val status: BuildStatus?
     val branch: Branch
+    val state: BuildState
+    val name: String
+    val canceledInfo: BuildCanceledInfo?
 
     /**
      * Web UI URL for user, especially useful for error and log messages
      */
-    fun getWebUrl(): String
+    fun getHomeUrl(): String
 
-    fun fetchStatusText(): String
-    fun fetchQueuedDate(): Date
-    fun fetchStartDate(): Date
-    fun fetchFinishDate(): Date
+    val statusText: String?
+    val queuedDate: Date
+    val startDate: Date?
+    val finishDate: Date?
 
-    fun fetchParameters(): List<Parameter>
+    val parameters: List<Parameter>
 
-    fun fetchRevisions(): List<Revision>
+    val revisions: List<Revision>
```

Comment:
What does `revisions` mean? The latest revisions from VCS roots attached to this build configuration? It makes sense to add KDoc here and maybe rename the property.

Reply:
Added kdocs for revisions and changes